src/main/java/com/ldtteam/structurize/blocks/MultiBlock.java

```diff
@@ -31,7 +31,7 @@
     /**
      * This blocks name.
      */
-    private static final String BLOCK_NAME = "multiBlock";
+    private static final String BLOCK_NAME = "multiblock";
```

Comment:
won't that break it?

Reply:
somewhere it's lowercased, somewhere with big B
probably yes

Reply:
Wait I just dig deeper,
`setRegistryName(Constants.MOD_ID.toLowerCase() + ":" + BLOCK_NAME);` was using big B
`GameRegistry.registerTileEntity(TileEntityMultiBlock.class, new ResourceLocation(Constants.MOD_ID, MultiBlock.getName()));` was using small b
So what was the actual registered name?

Reply:
SetRegistryname auto lowercases,
It should always be lowercase!

Reply:
yeah, I agree, but
setRegistryName only lowercases namespace, not the path
translationKey doesn't care
registerTileEntity use the same method for checking as setRegistryName (so only lowercases namespace)

Reply:
```java
int index = name.lastIndexOf(':');
String oldPrefix = index == -1 ? "" : name.substring(0, index).toLowerCase(Locale.ROOT);
name = index == -1 ? name : name.substring(index + 1);
```